website/docs/feature-flag-tutorials/golang/implementing-feature-flags-golang.md

```diff
@@ -0,0 +1,290 @@
+
+```go
+package main
+
+import (
+    "bytes"
+    "encoding/json"
+    "fmt"
+    "io"
+    "log"
+    "net/http"
+)
+
+type Country struct {
+    Name string `json:"name"`
+    Capital string `json:"capital"`
+}
+
+func main() {
+    // Define a static feature flag
+    isGraphQL := true
+
+    var country Country
+    if isGraphQL {
+        // Call the GraphQL API
+        query := `{"query": "query { country(code: \"NO\") { name capital } }"}`
+        req, err := http.NewRequest("POST", "https://countries.trevorblades.com/", bytes.NewBuffer([]byte(query)))
+        if err != nil {
+            log.Fatal(err)
+        }
+        req.Header.Set("Content-Type", "application/json")
+
+        client := &http.Client{}
+        resp, err := client.Do(req)
+        if err != nil {
+            log.Fatal(err)
+        }
+        defer resp.Body.Close()
+
+        body, _ := io.ReadAll(resp.Body)
+        var response struct {
+            Data struct {
+                Country Country `json:"country"`
+            } `json:"data"`
+        }
+        json.Unmarshal(body, &response)
+
+        country = response.Data.Country
+        fmt.Println("Hello GraphQL")
+    } else {
+        // Call the REST API
+        resp, err := http.Get("https://restcountries.com/v2/alpha/no")
+        if err != nil {
+            log.Fatal(err)
+        }
+        defer resp.Body.Close()
+
+        body, _ := io.ReadAll(resp.Body)
+        var countries []Country
+        json.Unmarshal(body, &countries)
+        country = countries[0]
+    }
+
+    fmt.Printf("Country: %s, Capital: %s\n", country.Name, country.Capital)
+}
+```
+
+Run the code again:
+
+```sh
+go run main.go
+```
+
+You should see `Hello GraphQL`, followed by `Country: Norway, Capital: Oslo` in your terminal.
+
+## 5. Add Unleash to your Go app
+
+Now, let's connect our project to Unleash so that you can toggle that feature flag at runtime. If you wanted to, you could also do a gradual rollout, use it for a/b testing, etc.
+
+You'll need 2 things:
+
+-   The URL of your Unleash instance's API. It's `http://localhost:4242/api/` for your local version. You'll want to replace it with your remote instance.
+-   The API token we created on our Unleash instance, feel free to create another one if you can't find it.
+
+With these 2, you can initialize your Unleash client as follows:
+
+```go
+unleash.Initialize(
+    unleash.WithUrl("http://localhost:4242/api/"),
+    unleash.WithAppName("country_go"),
+    unleash.WithCustomHeaders(http.Header{"Authorization": {"YOUR_API_KEY"}}),
+)
+```
+
+You can check our [API token and client keys documentation](https://docs.getunleash.io/reference/api-tokens-and-client-keys) for more specifics.
+
+Now, let's add our client to our project, grab the feature flag from Unleash, and update our conditional statement. Don't forget to also update the config with your API key:
+
+```diff
```

Comment:
How is this supposed to render? I don't think it works at the moment

Reply:
You're right! This was a problem with `diff` blocks across the board, fixed it 😄 https://github.com/Unleash/unleash/pull/8937